app/src/main/res/values-en/strings.xml

```diff
@@ -1236,4 +1236,134 @@
     <string name="permissions_required">Required permissions</string>
     <string name="settings_register_hide_sticks_from_old">Your mother won\'t see your F grades</string>
+    <string name="build_official">Official build</string>
+    <string name="build_platform_play">Google Play</string>
+    <string name="login_chooser_testing_title">Test version</string>
+    <string name="login_chooser_mode_recommended">{cmd-information-outline} Recommended</string>
+    <string name="login_chooser_mode_dev_only">{cmd-android-studio} Developer version</string>
+    <string name="login_chooser_mode_testing">{cmd-alert-circle-outline} Test version</string>
+    <string name="eggs">\???</string>
+    <string name="see_also">See also</string>
+    <string name="settings_about_github_text">Source code</string>
+    <string name="build_branch">Git branch</string>
+    <string name="build_platform_apk">.APK</string>
+    <string name="build_platform_unofficial">Unofficial (.APK)</string>
+    <string name="build_commit">Commit hash</string>
+    <string name="build_dirty">Unsaved changes</string>
+    <string name="build_tag">Last build tag</string>
+    <string name="build_rev_count">Commits from last build tag</string>
+    <string name="build_remote">Remote repository</string>
+    <string name="build_details">Build details</string>
+    <string name="build_dialog_open_repo">Check code</string>
+    <string name="profile_config_name_hint">Profile name</string>
+    <string name="profile_config_logout">Logout</string>
+    <string name="profile_config_sync_enabled">Synchronize this profile</string>
+    <string name="home_availability_update">Update</string>
+    <string name="update_available_later">Not now</string>
+    <string name="update_available_button">Update</string>
+    <string name="login_type_edudziennik">EduDziennik</string>
+    <string name="login_form_title_format">Log in - %s</string>
+    <string name="attendance_details_date">Date</string>
+    <string name="attendance_details_teacher">Teacher</string>
+    <string name="attendance_tab_days">By days</string>
+    <string name="attendance_tab_list">List</string>
+    <string name="attendance_lesson_number_format">lesson %d</string>
+    <string name="attendance_tab_types">By type</string>
+    <string name="settings_add_student_text">Add new student</string>
+    <string name="attendance_details_lesson_topic">Lesson subject</string>
+    <string name="register_unavailable_read_more">Read more</string>
+    <string name="build_version">App version</string>
+    <string name="settings_about_version_details_text">Version details</string>
+    <string name="build_date">Build date</string>
+    <string name="error_no_api_access">No API access</string>
+    <string name="attendance_details_time">Hour</string>
```

Comment:
Didn't notice this before, this should be Time because it also presents minutes.